modules/iam/README.md

```diff
@@ -21,5 +21,8 @@ module "cluster" {
   name       = "sal-9000"
   iam_config = module.iam.config
   ...
+  depends_on = [
```

Comment:
📝 We support terraform >=0.12
https://github.com/cookpad/terraform-aws-eks/blob/master/versions.tf#L2
and `depends_on` requires >=0.13